app/views/details/personal-information/immigration-status.html

```diff
@@ -7,62 +7,29 @@
 
 {% block beforeContent %}
   {{ govukBackLink({
-    href: (referrer if referrer else "/details/personal-information/immigration/")
+   href: "/details/personal-information/immigration"
   }) }}
 {% endblock %}
 
-{% block formBody %}
-  {% if isEuropeanCitizen == true %}
-    {{ govukRadios({
-      fieldset: {
-        legend: {
-          html: title,
-          classes: "govuk-fieldset__legend--l",
-          isPageHeading: true
-        },
-        attributes: {
-          "data-module": "clear-hidden"
-        }
-      },
-      items: [{
-        value: "EU settled status",
-        text: "EU settled status"
-      }, {
-        value: "EU pre-settled status",
-        text: "EU pre-settled status"
-      }, {
-        value: "other",
-        text: "Another immigration status",
-        conditional: {
-          html: govukCharacterCount({
-            label: {
-              text: "Immigration status"
-            },
-            hint: {
-              text: "For example, “Indefinite leave to remain”."
-            },
-            threshold: 90,
-            maxwords: 200,
-            rows: 5
-          })
-        }
-      }]
-    }) }}
-  {% else %}
-    {{ govukCharacterCount({
-      label: {
-        text: title,
-        classes: "govuk-label--l"
-      },
-      hint: {
-        text: "For example, “I have permanent residence”."
-      },
-      maxwords: 200,
-      rows: 5
-    }) }}
-  {% endif %}
+{% block content %}
+
```

Comment:
The width seems very long on these pages, think we're missing the two-thirds tags:
```
{% block content %}
  <div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">
...........
..........
.........
</div>
</div>

{% endblock %}
```